src/EditorFeatures/Core.Wpf/Suggestions/SuggestedActionsSourceProvider.cs

```diff
@@ -38,12 +38,14 @@ namespace Microsoft.CodeAnalysis.Editor.Implementation.Suggestions
     [Order]
     [SuggestedActionPriority(DefaultOrderings.Highest)]
     [SuggestedActionPriority(DefaultOrderings.Default)]
+    [SuggestedActionPriority(DefaultOrderings.Low)]
```

Comment:
this needs more work i believe.  teh AsyncSuggestedActionSource literally expects *only* 3 values to be passed in.  See  code like:

```using var _2 = ArrayBuilder<SuggestedActionSet>.GetInstance(out var lowPrioritySets);```

If we have more than 3 values, we'll need to beef this up to understand that we have things like Low/Lowest.  Specifically, we'll need a mapping from priority to the items found for that priority in a higher pass.  then we'll need to add those items to the right bucket when we get to it.

Reply:
this is also why i only added support for these three to begin with :)  because it was eitehr:

1. highest=high pri items. (rename and add-using)
2. default=everything else.  though some items might show up here that were generated by the high-pri item (For example, 'add reference to dll' can be generated by add-using).
3. lowest=suppression/config.

So basically it was just high/low, with a final bucket for config.  And there was only one way for items to waterfall down to the next bucket.

that made the impl very simple.  i have no problem with adding Low (or even High), but it def means the code needs to be updated to know it might have many different priorities and that waterfalling may have to happen between multiple levels.

Reply:
Thanks for the details @CyrusNajmabadi. Implemented this with https://github.com/dotnet/roslyn/pull/66970/commits/9ae979425625c7a1fc9dfa9ae3a24828653dd0dc